statbot/crawler.py

```diff
@@ -252,12 +252,12 @@ def __init__(self, client, sql, config, logger=null_logger):
         self.logger.info(f"Starting from ID {last_id} ({last})")
 
         entries = await guild.audit_logs(after=last, limit=limit).flatten()
```

Comment:
can you leave a comment about how `after=last` doesn't exclusively give you audit log entries after the one you request and thus the silly `!=` behavior?